app/assets/javascripts/controllers/container_deployment/dialogs.module.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * @name  MiQ Dialogs
+ *
+ * @description
+ *   Dialogs module for MiQ.
+ *
+ */
+angular.module('miq.dialogs', []);
```

Comment:
> I'm a bit concerned about this file's location .. or rather, the only file that defines stuff in this module is container_deployment/modal_dialog/modal-dialog-directive.js .. I mean, this should either be part of that file and not be called miq.dialogs but rather something more specific,

I tried to maintain consistency with the container dashboard modules and directive formats. for example you have `card-module.js` and it's directive `heatmaps-card-directive.js`. I don't mind changing the module name but we probably be consistent.

> or if it is called miq.dialogs and meant to contain all the miq dialogs, then it makes no sense to put it in container_deployment.. (but yeah, that's a detail at best..:) )

that wasn't my intention :)